tests/integration/annotation_import/conftest.py

```diff
@@ -27,11 +28,9 @@ def audio_data_row(rand_gen):
 def conversation_data_row(rand_gen):
     return {
         "row_data":
```

Comment:
using the same data as our notebooks.... I thought in the beginning this was one of the reasons for failing conversational export tests